packages/grafana-ui/src/components/uPlot/utils.ts

```diff
@@ -119,16 +119,16 @@ export function findMidPointYPosition(u: uPlot, idx: number) {
   for (let i = 1; i < u.data.length; i++) {
     const sData = u.data[i];
     const sVal = sData[idx];
-    if (sVal !== null) {
-      if (max === null) {
+    if (sVal != null) {
+      if (max == null) {
```

Comment:
can you add comments on why (and if) it is important that this is `==` not `===` many folks looking at it (including me a few weeks ago) have learned to just use `===` for some who knows what javascript reason

Reply:
this will (and already does) exist in many places. adding a comment here specifically seems odd. if TS does not complain then not sure it's worth also adding comments everywhere instead of expecting folks to know how JS works :)

Reply:
in fact, making this strict induces TS errors, so i dont think it's necessary.